frame/assets/rpc/src/lib.rs

```diff
@@ -0,0 +1,102 @@
+use assets_runtime_api::AssetsRuntimeApi;
+use codec::Codec;
+use jsonrpc_core::{Error as RpcError, ErrorCode, Result as RpcResult};
+use jsonrpc_derive::rpc;
+use sp_api::ProvideRuntimeApi;
+use sp_blockchain::HeaderBackend;
+use sp_runtime::{generic::BlockId, traits::Block as BlockT};
+use std::sync::Arc;
+
+#[rpc]
+pub trait AssetsApi<BlockHash, AssetId, AccountId, Balance> {
+	#[rpc(name = "assets_balanceOf")] // , returns = "Balance"
+	fn balance_of(
+		&self,
+		at: Option<BlockHash>,
+		currency: AssetId,
+		account: AccountId,
+	) -> RpcResult<Balance>;
+}
+
+/// A struct that implements the `AssetsApi`.
+pub struct Assets<C, Block> {
+	// If you have more generics, no need to Assets<C, M, N, P, ...>
+	// just use a tuple like Assets<C, (M, N, P, ...)>
+	client: Arc<C>,
+	_marker: std::marker::PhantomData<Block>,
+}
+
+impl<C, M> Assets<C, M> {
+	/// Create new `SumStorage` instance with the given reference to the client.
+	pub fn new(client: Arc<C>) -> Self {
+		Self { client, _marker: Default::default() }
+	}
+}
+
+/// Error type of this RPC api.
```

Comment:
please comment when going uncomment this code?

Reply:
removed